### PR TITLE
Enabling the monitor point export from the Icepak design settings.

### PR DESCRIPTION
**Pull request by TejasJe**
Contributing to fixing issue number #1912: Enabling the monitor point export from the Icepak design settings.

### CODE REVIEW

File: pyaedt/icepak.py [hunk 1]
```diff
@@ -1456,6 +1458,10 @@ def edit_design_settings(
             Default fluid material. The default is ``"air"``.
         defaultsolid : str, optional
             Default solid material. The default is ``"Al-Extruded"``.
+        export_monitor : bool, optional
+            Default export option for monitor point data. The default is ``"False"``
```
Comment (MaxJPRey) [inline pyaedt/icepak.py:1462]:
```suggestion
            Default export option for monitor point data. The default is ``False``
```

File: pyaedt/icepak.py [hunk 2]
```diff
@@ -1456,6 +1458,10 @@ def edit_design_settings(
...
+        export_directory : str, optional
+            Default export directory for monitor point data. The default is ``"current working directory"``.
```
Comment (MaxJPRey) [inline pyaedt/icepak.py:1464]:
```suggestion
            Default export directory for monitor point data. The default value is the current working directory.
```

**Comment by MaxJPRey** [comment]
@TejasJe Thanks for contributing and enabling the export of the monitor point!
Could you also add a unit test for these 2 new parameters?

**Comment by TejasJe** [comment]
@MaxJPRey Thank you for your suggestions Maxim. I have committed to the changes suggested by you. 
Let me add the unit tests.

**Comment by TejasJe** [comment]
@MaxJPRey Added the unit test as per your suggestion

File: pyaedt/icepak.py [hunk 3]
```diff
@@ -1456,6 +1458,11 @@ def edit_design_settings(
             Default fluid material. The default is ``"air"``.
         defaultsolid : str, optional
             Default solid material. The default is ``"Al-Extruded"``.
+        export_monitor : bool, optional
+            Whether to use the default export directory for monitor point data.
+            The default is ``False``
```
Comment (MaxJPRey) [inline pyaedt/icepak.py:1463]:
```suggestion
            The default value is ``False``.
```

**Review by MaxJPRey — APPROVED** [review]
LGTM. Thanks @TejasJe for contributing to the project.

**Pull request merged**